### ISSUE TITLE
error in _jstrans.js

### ISSUE THREAD

**Issue by davideloba** [emdemir/Flask-Babel-JS#2]
Hello,
there is an error in the _jstrans.js file: an extra ";" at line
`babel.plural = function(n) { return ((n != 1);); }`
compiled from 
`js.append(c2js(m.lower().split("plural=")[1]))`

Flask-Babel==1.0.0
Flask-Babel-Js==1.0.2

**Comment by demorose** [comment]
I had the same issue.

It was due to my po file. I used poedit to edit it, and it added a semi-colon to "Plural-Forms".
I removed it and it work as intended now.

**Comment by LightDashing** [comment]
Same problem, tried to edit .po file with Poedit and Lokalize, same results in both programs. As far as I understand, when .po file compiles into .mo version, compiler adds semicolon at the end of plural=...
If you open .mo version with some text editor, you will see something like this:
```
Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);
```

**Comment by TheKruspe** [comment]
I created a [PR](#5)  with the suggested fix from @solsticedhiver.

**Comment by bbird81** [comment]
I have the same problem... no correction after 2 years??

**Comment by emdemir** [comment]
Fixed with #5. Sorry for not taking a look for a long time.

**Comment by emdemir** [comment]
@BornSupercharged @bbird81 I just released 1.0.3 to PyPI. Could you please test and let me know? I currently don't have the means to set up a test project for this (see #9 for future plans once I do).

**Comment by BornSupercharged** [comment]
@emdemir yes 1.0.3 works properly thank you!
